docs/blog/2019-03-25-behind-the-scenes-what-makes-gatsby-great/index.md

```diff
@@ -0,0 +1,496 @@
+      /static/55558/your-great-image.jpg 1200w,
+      /static/ca53f/your-great-image.jpg 1600w,
+      /static/2e134/your-great-image.jpg 2400w" sizes="(max-width: 800px) 100vw, 800px">
+    <img alt="" src="/static/3be68/your-great-image.jpg" style="position: absolute; top: 0px; left: 0px; width: 100%; height: 100%; object-fit: cover; object-position: center center; opacity: 1; transition: opacity 0.5s ease 0s;">
+  </picture>
+  <!-- highlight-end -->
+  <!-- JavaScript disabled? No problem. -->
+  <noscript>
+    <picture>
+      <source srcSet="/static/aa813/your-great-image.jpg 200w,
+        /static/d7872/your-great-image.jpg 400w,
+        /static/3be68/your-great-image.jpg 800w,
+        /static/55558/your-great-image.jpg 1200w,
+        /static/ca53f/your-great-image.jpg 1600w,
+        /static/2e134/your-great-image.jpg 2400w" sizes="(max-width: 800px) 100vw, 800px" />
+      <img src="/static/3be68/your-great-image.jpg" alt=""style="position:absolute;top:0;left:0;transition:opacity 0.5s;transition-delay:0.5s;opacity:1;width:100%;height:100%;object-fit:cover;object-position:center"/>
+    </picture>
+  </noscript>
+</div>
+```
+
+For a demo of all you can do with `gatsby-image`, check out the example [Using Gatsby Image](https://using-gatsby-image.gatsbyjs.org).
+
+The power and appeal of Gatsby is that we internalize all of these great APIs, performance techniques, and patterns. Enabling these (and improving with incremental releases of Gatsby!) allow you to focus on what matters: delivering compelling and engaging user experiences backed by your friendly, fast, and powerful web compiler: Gatsby.
+
+It's been quite a ride 😅 By this point, we've covered the foundational techniques Gatsby uses to deliver a performance optimized application for your end users. However--to assert something is fast, you need to measure it. You need analytics, tools, and checks to assert that not only is your app fast but also just as important that it _remains_ fast.
+
+## Techniques for measuring performance
+
+With any application, even one with a rock-solid performance basis like Gatsby, the advice I'd give is to trust, but validate. It's _unlikely_ you'll introduce a significant performance regression with a new feature and pull request, but even the best of foundations can be torn down. "I think I'll load _six_ fonts each in weights 100-900" he said and "Oh, and I think two feedback libraries _and_ a few analytics libraries are really going to drive conversion and put us over the edge," said someone, assuredly, at some point.
+
+### Lighthouse
+
+The quickest and easiest recommendation for measuring performance is to first and foremost use the tools available to you. Chrome has an excellent tool built-in, [Lighthouse](https://github.com/GoogleChrome/lighthouse) accessible via the "Audits" tab in Developer Tools (and fun fact--Gatsby's very own [Ward Peeters](https://github.com/wardpeet) is a core contributor!).
```

Comment:
I've also been a core contributor to the underlying accessibility API where issues are sometimes filed, not sure if that's worth mentioning :)

Reply:
Of course it is!! Thanks for letting me know--will add you 👍

Reply:
Did add you! This OK?

> and fun fact--Gatsby's very own [Ward Peeters](https://github.com/wardpeet) and [Marcy Sutton](https://twitter.com/marcysutton) are core contributors!